modules/oc-mirror-support.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="oc-mirror-support_{context}"]
+= oc-mirror compatibility and support
+
+The oc-mirror plug-in supports mirroring {product-title} payload images and Operator catalogs for {product-title} versions 4.9 and later.
+
+Use the latest available version of the oc-mirror plug-in regardless of which versions of {product-title} you need to mirror.
+
+// TODO: remove this note for 4.12
```

Comment:
I was thinking for a minute on this and maybe I am overcomplicating, but would it be easier for you later if you made the above changes in main and then add the important note in a separate PR for 4.11? I guess you could do it either way though.

Reply:
Yeah true probably would have been, but I think it'll require 2 PRs anyways so doesn't matter too much